python/pyspark/streaming/dstream.py

```diff
@@ -602,13 +602,17 @@ class TransformedDStream(DStream):
     Multiple continuous transformations of DStream can be combined into
     one transformation.
     """
-    def __init__(self, prev, func):
+    def __init__(self, prev, func, transformFunc=None):
```

Comment:
Previously I use override to do it, but have several problem with `TransformFunction`/`TransformDStream` deserialization problem. I guess it is the derived class which makes deserialization error, so instead of using `KafkaTransformFunction`/`KafkaTransformDStream`, I added one another field to workaround this problem. I admitted this implementation is weird, do you have any better suggestion?